docs/library/index.rst

```diff
@@ -0,0 +1,17 @@
+Libraries
+===============================================================================
+
+.. note::
+
+   This page lists only the IPs that have been ported to the new documentation
+   format.
+
+Contents
+-------------------------------------------------------------------------------
+
+.. toctree::
+   :maxdepth: 1
+
+   frameworks/index
+   IPs/index
+   xilinx/index
```

Comment:
Why the need for grouping frameworks, IPs and xilinx separately? I would keep them "flattened" as in the hdl repo.

Reply:
Removed those subfolders, moved everything under library.